md/datetimes-management-tutorial.md

```diff
@@ -0,0 +1,163 @@
+
+You can now run your process and see that the date retrieved from the business data database is consistent with the date you chose in the previous step.
+
+## Date and time without time zone
+
+The type **DATE-TIME (NO TZ)** type is for manipulating a date and a time which are absolute. Meaning that the time displayed should be the same regardless of the user time zone (e.g., opening hours of a store somewhere, or local departure time for a flight).  
+The storage format for this type is an ISO 8601 formatted Character String (e.g. "2017-01-13T13:00:00").  
+
+### Define your business data
+
+* In the studio, start by opening the business data model editor (Menu "Business Data Model" > "Manage...")
+* Create an object with an attribute of type DATE-TIME (NO TZ)
+* Create a new process
+* Then, in the process data tab, create a new business variable of the type of your business object
+
+### Define the contract and task operations or business variable initialization script
+
+* In the process/task execution tab > contract section, define the contract inputs using the Add from data... button
+* Select the business variable you created previously
+* Follow the steps to generate a contract with a COMPLEX input containing a DATE-TIME (NO TZ) attribute (keep the option to generate the variable initialization script or the operations (for a task) automatically)
+
+### Generate the form
+
+* Generate the UI designer form by clicking on edit in the form section of the process/task execution tab
+* Since you have a DATE-TIME (NO TZ) attribute in the contract inputs, a date time picker widget is automatically added to the form. It is configured not to handle the user time zone  
+There is nothing else to do.
+
+::: info
+The date time picker widget output is an ISO 8601 formatted String (e.g. "2017-01-13T15:00:00"). At contract level when received by the process instantiation / task execution REST API, it is converted to a `java.time.LocalDateTime`.
+:::
+
+### Retrieve the date from the database and display it
+
+In another step form (or in any other page) you may want to retrieve the date and display it.  
+In order to do that:  
+* Create a new task after the existing one
+* Generate its form by clicking on edit in the form section of the task execution tab
+* In the UI Designer, you need first to define a new variable of type external API to retrieve the object containing the date attribute using the context:
+
+![REST API request](images/dates-management-tutorial/tuto-dates-api-request.png) <!--{.img-responsive .img-thumbnail}-->
+
+::: info
+The value returned by the REST API will be an ISO 8601 formatted String.  
+:::
+
+* Drop a date time picker widget in the page and configure it not to handle time zone
```

Comment:
1. Drop a **DATE TIME PICKER** widget onto the form
2. Set the property **Handle time zone** to _no_.